docs/api/workflows/projects.md

```diff
@@ -48,6 +48,9 @@ EOF
 ```
 
 Setting the `field.cattle.io/creatorId` field allows the cluster member account to see project resources with the `get` command and view the project in the Rancher UI. Cluster owner and admin accounts don't need to set this annotation to perform these tasks.
+Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```

Comment:
```suggestion

Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```

Adding newline for consistency with the rest of the doc. Right now line 50 and 51 renders as continuous line.